recipes/pypardiso/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "pypardiso" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/haasad/PyPardisoProject/archive/refs/tags/v0.2.2.tar.gz
+  sha256: b1d3a139f8f04c74f8b2b5c4b55434e1f58b00ae3fe0ca975115971906d93eac
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<35]
+
+requirements:
+  build:
+    - python
+    - setuptools
+  host:
+    - python
+    - setuptools
+    - numpy
+    - pip
+    - "libblas =*=*mkl"  # [win]
+    - mkl 2020.*  # [win]
```

Comment:
```suggestion
```

Reply:
This is necessary for the builds to work properly. See https://github.com/conda-forge/staged-recipes/pull/15106/commits/1ab96ed6cec87034da962e013767caa5eb2d6a03 and failures on previous versions.